docs.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+MODIFIED_FILES=$(git --no-pager diff --name-only ${TRAVIS_COMMIT_RANGE})
+if grep -q "docs/" <<<$MODIFIED_FILES
+then
+  mkdir out;
+  cd out
+  git clone https://github.com/ParsaLab/cloudsuite.git
+  cd cloudsuite
+  git checkout gh-pages
+  git config user.name ${GIT_USER}
+  git config user.email ${GIT_EMAIL}
+  git config credential.helper "store --file=.git/credentials"
+  git config --global push.default matching
+  echo "https://$GH_TOKEN:x-oauth-basic@github.com" >> .git/credentials
+  if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ]
+  then
+    git merge master --no-edit
```

Comment:
The merge should happen if there were no misspellings!
So it should be after checking the spellings.